unrpyc.py

```diff
@@ -283,29 +282,12 @@ def main():
 
     ap.add_argument(
         '-t',
-        '--translation-file',
-        dest='translation_file',
-        type=Path,
-        action='store',
-        default=None,
-        help="Use the specified file to translate during decompilation")
-
-    ap.add_argument(
-        '-T',
-        '--write-translation-file',
-        dest='write_translation_file',
-        type=Path,
-        action='store',
-        default=None,
-        help="Store translations in the specified file instead of decompiling")
-
-    ap.add_argument(
-        '-l',
-        '--language',
-        dest='language',
+        '--translate',
+        dest='translate',
+        type=str,
         action='store',
-        default=None,
-        help="If writing a translation file, the language of the translations to write")
+        help="Switches the apps main language to the the given language. This string must be "
+        "present in the tl dir and a exact match.")
```

Comment:
This isn't very clear on what it does. The original also wasn't, but this doesn't really improve much. 

It changes the language of the dialogue in the decompiled script files, using translations found in the script files, or something like that would be better.

Reply:
OK